open_squirrel/utils/identity_filter.py

```diff
@@ -1,4 +1,4 @@
-from opensquirrel.squirrel_ir import Gate
+from open_squirrel.ir import Gate
 
 
 def filter_out_identities(l: [Gate]):
```

Comment:
I don't know if we still need an identity filter, after our chat with Richard, he mentioned he did want explicit identities. 

However, this needs be in place when we decompose.

Reply:
No idea about this. Does this have to do with the Identity gate being added to OpenSquirrel?